TAO/TAO_IDL/util/utl_global.cpp

```diff
@@ -1953,7 +1953,7 @@ namespace
   {
   public:
     explicit OldState (bool disable_output = false)
-        : old_filename_ (idl_global->filename ()),
+        : old_filename_ (idl_global->filename () ? new UTL_String (idl_global->filename (), true) : 0), // need a copy because IDL_GlobalData::set_filename() destroys previous value
```

Comment:
break up long line

Reply:
What is the recommended maximum number of characters? It's not easy to break that line very nice - I'll try to break at the comment. Let me know how it looks.

Reply:
The official guidelines are 80 characters but as long as it's reasonably close to that...